File: tools/bench/answers/python_sonnet/s2/08_invoice.py
items = [("widget", 4, 2.5), ("gizmo", 2, 7.25), ("bolt", 10, 0.1)]
grand_total = 0
best_name = ""
best_total = -1
for name, qty, price in items:
    line_total = qty * price
    grand_total += line_total
    if line_total > best_total:
        best_total = line_total
        best_name = name
print(grand_total)
print(best_name)

File: tools/bench/answers/go_sonnet/s1/08_invoice.go
package main

import "fmt"

func main() {
	type item struct {
		name  string
		qty   float64
		price float64
	}
	items := []item{
		{"widget", 4, 2.5},
		{"gizmo", 2, 7.25},
		{"bolt", 10, 0.1},
	}
	total := 0.0
	maxTotal := 0.0
	maxName := ""
	for _, it := range items {
		lineTotal := it.qty * it.price
		total += lineTotal
		if lineTotal > maxTotal {
			maxTotal = lineTotal
			maxName = it.name
		}
	}
	fmt.Println(total)
	fmt.Println(maxName)
}
